powershell/enterprisePolicies/README.md

```diff
@@ -0,0 +1,150 @@
+# Power Platform Enterprise Policies PowerShell Scripts
+
+These scripts introduces the capabilities of the Enterprise Policies ARM resources. The sample scripts provides capabilities to create, update, delete and get enterprise policies.  
+It also provides capabilities to apply the enterprise policies to environments.
+
+## How to run setup scripts
+
+1. **Install module script** : This script installs the required module to run Enterprise Policies scripts.</br>
+Script name: InstallPowerAppsCmdlets.ps1</br>
+Run the script to import required PowerShell modules.
+
+2. **Setup Azure subscription for Microsoft.PowerPlatform** : This script registers the Azure subscription for Microsoft.PowerPlatform resource provider and also allow lists the subscription for enterprisePoliciesPreview feature.</br>
```

Comment:
I don't understand this second part. What does "allow lists the subscription" mean?

Reply:
Enterprise Policy resource is behind a feature gate in ARM called "enterprisePoliciesPreview". To work with enterprise policies the subscription should be allow listed for this feature in ARM.